sw/device/tests/crypto/BUILD

```diff
@@ -226,6 +227,73 @@ opentitan_functest(
         "//sw/device/lib/crypto/impl:integrity",
         "//sw/device/lib/crypto/impl:rsa",
         "//sw/device/lib/runtime:log",
+        "//sw/device/lib/testing:profile",
+        "//sw/device/lib/testing/test_framework:ottf_main",
+    ],
+)
+
+opentitan_functest(
+    name = "rsa_3072_keygen_functest",
+    srcs = ["rsa_3072_keygen_functest.c"],
+    cw310 = cw310_params(
+        timeout = "long",
+    ),
+    verilator = verilator_params(
+        tags = ["broken"],  # This test is too slow to run on verilator.
+    ),
```

Comment:
If you don't want to run it on Verilator, I'm guessing you also don't intend this for DV? If so, you should just set  [`target`](https://cs.opensource.google/opentitan/opentitan/+/master:rules/opentitan_test.bzl;drc=2b26b44c9a1c1bdf92a4c28e4e550470c32ecd34;l=332)` = `[`FPGA_DEFAULT_TARGETS`](https://cs.opensource.google/opentitan/opentitan/+/master:rules/opentitan_test.bzl;drc=2b26b44c9a1c1bdf92a4c28e4e550470c32ecd34;l=46), rather than marking verilator as `broken`.

Reply:
applies throughout

Reply:
Looks like `FPGA_DEFAULT_TARGETS` has been replaced as of a few days ago; I updated it to target FPGA-only following the new `exec_env` setup.

Reply:
ah nice, yes using the new bazel rules is even better :)